src/styles/theme.scss

```diff
@@ -92,4 +92,20 @@ $line-height-2:54px;
 // EXAMPLE
 .preamble {
   @extend .normal-text;
-}
\ No newline at end of file
+}
+
+// layout classes
+
+.flex-space-between{
+  display: flex;
+  justify-content: space-between;
+
+}
+
+.flex-center{
+  display: flex;
+  justify-content: center;
+}
+
+// standard borders
+$section-separator: 1px solid $black-haze;
```

Comment:
Consider moving this variable definition into an area in the file where other variable definitions are. Then, below the variable definitions, place any classes, such as the flex classes, that you'll use to extend styles in other classes.